style.json

```diff
@@ -230,15 +230,172 @@
         ]
       }
     },
+    {
+      "id": "waterway-bridge-case",
+      "type": "line",
+      "source": "openmaptiles",
+      "source-layer": "waterway",
+      "filter": [
+        "all",
+        [
+          "==",
+          "$type",
+          "LineString"
+        ],
+        [
+          "==",
+          "brunnel",
+          "bridge"
+        ]
+      ],
+      "layout": {
+        "line-cap": "butt",
+        "line-join": "miter"
+      },
+      "paint": {
+        "line-color": "#dedede",
```

Comment:
Please consider some darker color, e.g. `#bbb` (used in the image above). With `#dedede` the casing is not quite visible over residential landuse.